lib/runtime/test_helpers.go

```
@@ -280,3 +282,24 @@ func (trs testRuntimeStorage) GetBalance(key [32]byte) (uint64, error) {
 
 	return binary.LittleEndian.Uint64(bal), nil
 }
+
+type testRuntimeNetwork struct {
+}
+
+func (trn testRuntimeNetwork) NetworkState() common.NetworkState {
+	testAddrs := []ma.Multiaddr(nil)
+
+	// create mock multiaddress
+	addr, err := ma.NewMultiaddr("/ip4/127.0.0.1/tcp/7001/p2p/12D3KooWDcCNBqAemRvguPa7rtmsbn2hpgLqAz8KsMMFsF2rdCUP")
+	if err != nil {
+		// todo ed handle error
+		fmt.Printf("ERROR %v\n", err)
+	}
```

you can just ignore the error here